.github/CODEOWNERS

```diff
@@ -59,6 +59,7 @@
 # TODO: if we ever add `.internal` API reports, we will want to omit them here
 /**/api-report/*.api.md                        @microsoft/fluid-cr-api
 /build-tools/**/api-report/*.api.md            # Do not require API review for build-tools packages
+/packages/framework/presence/**/api-report/*.api.md # Do not require API review for experimental presence package
```

Comment:
rather than this, can you just turn off api docs generation in the presence package?

Reply:
I could, but I prefer seeing what is really changing if anything in the report.